docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+
+More customization can be done to increase the efficiency of global and detail route. Refer to the [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/).
+
+
+## Metal Tracks Configuration <a name="MetalTracksConfiguration"></a>
+
+OpenROAD requires a metal track configuration file for use in floorplanning. For each metal layer, the x and y offset as well as the x and y pitch are defined. To find the pitch and offset for both x and y, refer to the tech LEF. Following is a generalized metal tracks configuration file with five metal tracks defined.
+
+
+```
+# make_tracks.tcl
+###############################
+make_tracks metal1 -x_offset 0.24 -x_pitch 0.82 -y_offset 0.24 -y_pitch 0.82
+make_tracks metal2 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal3 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal4 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal5 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+```
+
+
+Refer to the tech file to determine the x and y pitch and the offset.
+
+
+## PDN Configuration <a name="PDNConfiguration"></a>
+
+PDN is a utility that simplifies adding a power grid into the floorplan. With specifications given in the PDN configuration file, like which layer to use, stripe width and spacing, the utility can generate the metal straps used for the power grid. To create and configure a power grid, refer to [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/) on pdngen. Following is a generic PDN configuration file:
+
+
+```
+# pdn.cfg
+###############################
+# POWER or GROUND #Std. cell rails starting with power or ground rails at the bottom of the core area
+set ::rails_start_with "POWER" ;
+
+# POWER or GROUND #Upper Mal stripes starting with power or ground rails at the left/bottom of the core area
+set ::stripes_start_with "POWER" ;
+
+# Power nets
+set ::power_nets "VDD"
+set ::ground_nets "VSS"
+
+set pdngen::global_connections {
+VDD {
+  {inst_name .* pin_name VPWR}
+  {inst_name .* pin_name VPB}
+}
+VSS {
+  {inst_name .* pin_name VGND}
+  {inst_name .* pin_name VNB}
+}
+}
+#===> Power grid strategy
+# Ensure pitches and offsets will make the stripes fall on track
+pdngen::specify_grid stdcell {
+  name grid
+  rails {
+      M1 {width 1.04 offset 0}
+  }
+  straps {
+      M4 {width 2.08 pitch 20.28 offset 0}
+      M5 {width 2.08 pitch 20.28 offset 0}
+  }
+  connect {{M1 M4} {M4 M5}}
+}
+
+pdngen::specify_grid macro {
+  orient {R0 R180 MX MY}
+  power_pins "VPWR"
+  ground_pins "VGND"
+  blockages "M1 M2 M3 M4"
+  connect {{M4_PIN_ver M5}}
+}
+
+pdngen::specify_grid macro {
+  orient {R90 R270 MXR90 MYR90}
+  power_pins "VPWR"
+  ground_pins "VGND"
+  blockages "M1 M2 M3 M4"
+  connect {{M4_PIN_hor M5}}
+}
+```
```

Comment:
Wouldn't it be better to use the tcl interface for PDN?

Reply:
Removed sample script and instead referenced to PDN documentation for the tcl interface.